.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 dist
 .DS_Store
-.idea
+.idea
```

Comment:
This is not a big deal but usually I'd prefer new line at end file

Reply:
Oh sorry, please help me to add a new line at end file @lqt93